repo/src/main/amp/config/alfresco/module/inform-policy/context/bootstrap-context.xml

```diff
@@ -8,18 +8,22 @@
 		 workflow, and loading custom content models.  -->
 
     <!-- Bootstrap -->
-    <bean id="com.alvexcore.inform-bootstrap"
-          parent="spacesStoreImporter">
-        <property name="useExistingStore">
-            <value>false</value>
+    <bean id="com.alvexcore.InformBootstrapPatch"
```

Comment:
Please, rename InformBoostrap to InformPolicyBootstrap or something like this, that includes full component name.